spec/factories/fleet_trucks.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :fleet_truck do
+    status "MyString"
```

Comment:
You need to ensure that the status is one of the expected value (EG: "standard" or "auxiliary").

You could do something like

```ruby
status { ["standard", "auxiliary"].sample }